iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -324,6 +332,12 @@ public void rollbackCreateTable(org.apache.hadoop.hive.metastore.api.Table hmsTa
 
   @Override
   public void commitCreateTable(org.apache.hadoop.hive.metastore.api.Table hmsTable) {
+    String catalogType = MetastoreConf.get(conf, MetastoreConf.ConfVars.HIVE_ICEBERG_CATALOG_TYPE.getHiveName());
+    if (catalogType != null && !catalogType.isEmpty()) {
+      // If the catalog type is set, we assume that the table has been created in the delegate.createTable(request) call
```

Comment:
how was it working with before?

Reply:
Before my integration test `TestHiveRESTCatalogClientIT` worked with the iceberg hook, after adding the hook I found an issue in this place.

Reply:
what about other commits like `commitAlterTable`, `commitDropTable`, `commitInsertTable`? we should probably skip them as well
would be nice to have some generic solution, maybe even create diff hook?